balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -85,10 +76,6 @@ class MetricsServlet extends JacksonJsonServlet
       contentType = response.contentType
       response.result
     }
-    finally
-    {
-      periodMeter.update(System.currentTimeMillis() - begin, TimeUnit.MILLISECONDS)
```

Comment:
I believe I understand the source of my confusion-- I misunderstood what was going on in the syntax of the scalatra timer blocks (I didn't realize they were running code in-place rather than on a callback like the `get` blocks. It made sense when I remembered that {} is just the syntax for passing a closure that takes no arguments, and that it's not some weird method declaration syntax or anything more complicated). I should be able to sort it out pretty easily now.